lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -2434,5 +2434,124 @@ def test_cell_method_correct_order(self):
         self.assertTrue(cube1 == cube2)
 
 
+class Test__summary_dim_name(tests.IrisTest):
```

Comment:
Need a test for the [multiple dimension coords mapping `ValueError`](https://github.com/SciTools/iris/pull/3914/files#diff-38eaa350bf479c486e9f6659b8077be2d52448f17739872e50285b98b5423415R2280-R2288).

Reply:
I don't think so.  That is just defensive programming : logically it can never happen unless the Cube is "broken".